builder/builder.go

```diff
@@ -81,7 +81,7 @@ func NewBuilder(sk *bls.SecretKey, ds flashbotsextra.IDatabaseService, relay IRe
 		builderPublicKey:     pk,
 		builderSigningDomain: builderSigningDomain,
 
-		limiter:       rate.NewLimiter(rate.Every(time.Millisecond), 510),
+		limiter:       rate.NewLimiter(rate.Every(time.Millisecond), 600),
```

Comment:
I think 510 is enough above 500 to avoid issues most of the time for relays wanting 2 blocks/s
I think the bigger problem we could try to solve is that some relays limit to 1 block/s and we'd want the rate limit to be configurable in that case